task/fbc-validation/0.1/fbc-validation.yaml

```
@@ -153,11 +155,11 @@ spec:
           exit 0
         fi
         # copy content of conffolder to confdir - will be used in next task - related image check
-        cp -r .$conffolder/* $(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir
+        cp -r ".$conffolder/*" "$(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir"
```

```suggestion
        cp -r ".$conffolder"/* "$(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir"
```
The globbing was intentional I think, so it has to stay outside the quotes

---

Thanks for pointing it out. addressed this in the latest commit `de28fc075885bb61f13f81196c55691b3793163e`